pages.it/common/git-clone.md

```diff
@@ -0,0 +1,24 @@
+# git clone
+
+> Clona un repository esistente.
+> Maggiori informazioni: <https://git-scm.com/docs/git-clone>.
+
+- Clona un repository remoto esistente:
+
+`git clone {{url_repository_remoto}}`
```

Comment:
The original version uses the term "location". However, I thought URL is more general and compliant to the official manuals (e.g., https://git-scm.com/book/it/v1/Basi-di-Git-Repository-Git)